start

```diff
@@ -114,6 +119,9 @@ function process_args() {
     --enable-soroban-diagnostic-events)
       ENABLE_SOROBAN_DIAGNOSTIC_EVENTS=true
       ;;
+    --enable-soroban-admin-endpoint)
+      ENABLE_SOROBAN_ADMIN_ENDPOINT=true
```

Comment:
This admin endpoint is for the soroban-rpc, so we should be explicit about that.
```suggestion
    --enable-soroban-rpc-admin-endpoint)
      ENABLE_SOROBAN_RPC_ADMIN_ENDPOINT=true
```